Entities/Industry/Saw/Saw.as

```diff
@@ -238,6 +238,33 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid)
 		Vec2f bpos = blob.getPosition();
 		this.Tag("sawed");
 		this.server_Hit(blob, bpos, bpos - pos, 0.0f, Hitters::saw);
+
+		if (blob.getName() == "bomb")
+		{
+			if (this.get_u8("bombs_exploded") == 0)
+			{
+				this.set_u32("bomb_time", getGameTime());
+			}
+
+			this.add_u8("bombs_exploded", 1);
+		}
+	}
+}
+
+void onTick(CBlob@ this)
```

Comment:
This can be done without having to use onTick, its not a big deal but imo if we can avoid using it, it would be best